.github/workflows/update_dependencies.yml

```
@@ -0,0 +1,37 @@
+name: Update dependencies
+
+on:
+    workflow_dispatch:
+    schedule:
+        # at 00:00 on Monday (UTC)
+        - cron: "0 0 * * MON"
+
+jobs:
+    update_dependencies:
+        runs-on: ubuntu-latest
+        steps:
+            - name: "Checkout"
+              uses: actions/checkout@v2
+              with:
+                  fetch-depth: 1
+            - name: Set up Python 3.8
+              id: py
+              uses: actions/setup-python@v2
+              with:
+                  python-version: 3.8
+                  cache: pip
+                  cache-dependency-path: "requirements/*.txt"
+            - name: Update dependencies
+              run: make update-deps
+            - name: Commit changes and open PR
+              run: |
+                  git config user.name github-actions
+                  git config user.email github-actions@github.com
+                  git switch -c "dependency-updates-week-${date +%u}"
+                  git add requirements/*.txt 
+                  git commit -m "update dependencies (automated)"
+                  git push origin "dependency-updates-week${date +%u}"
```

±: Syntax is slightly different:
```suggestion
                  export WEEK="W$(date +%V)"
                  git switch -c "dependency-updates-week-$WEEK"
                  git add requirements/*.txt 
                  git commit -m "$WEEK dependency updates (automated)"
                  git push origin "dependency-updates-week-$WEEK"
```

(Adding `$WEEK` to the commit message, since it'll be used as-is by `gh` in the next step.)